File: python/main/static_array.py
class StaticArray:
    def __init__(self, numbers):
        self.numbers = numbers

    def find(self, value):
        returnValue = False

        for item in self.numbers:
            if item == value:
                returnValue = True
                break

        return returnValue

    def print(self):
        for index, value in enumerate(self.numbers, start=0):
            print(f"numbers[{index}]={value}")

File: typescript/src/main/StaticArray.ts
export class StaticArray {
    private numbers: number[];

    constructor(numbers: number[]) {
        this.numbers = numbers;
    }

    public find(value: number): boolean {
        let found = false;

        for (let index = 0; index < this.numbers.length; index++) {
            if (value == this.numbers[index]) {
                found = true;
                break;
            }
        }

        return found;
    }

    public print() {
        this.numbers.forEach((item, index) => {
            console.log(`numbers[${index}=${item}]`);
        });
    }
}